README.md

```diff
@@ -117,3 +117,38 @@ See the [Ansible Community Guide](https://docs.ansible.com/ansible/latest/commun
 GNU General Public License v3.0 or later.
 
 See [LICENSE](https://www.gnu.org/licenses/gpl-3.0.txt) to see the full text.
+<!--start collection content-->
+## Cliconf plugins
+Name | Description
+--- | ---
+[cisco.iosxr.iosxr](https://github.com/ansible-collections/cisco.iosxr/blob/master/docs/cisco.iosxr.iosxr.rst)|Use iosxr cliconf to run command on Cisco IOS XR platform
```

Comment:
URL for cliconf plugin docs and netconf plugin docs are the same. It needs to be updated manually.